app/controllers/v2/rota_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class V2::RotaController < ApplicationController
+  def show
+    respond_to do |format|
+      format.ics { render plain: Patterdale::ICal.new(rota).results, mime_type: Mime::Type.lookup("text/calendar") }
```

Comment:
Do we need to return an iCal version for v2? Or shall we just keep v2 as JSON for now?

Reply:
I'd rather keep it as JSON only for now.

Reply:
No, I totally agree, so we probably don't need the line above 👍